src/is_face_detector/service.py

```diff
@@ -1,5 +1,6 @@
 import re
 import dateutil.parser as dp
+from urllib.parse import urlparse
```

Comment:
where are u using it?

Reply:
hihihihi sorry :)

I was trying to use the urlparse, but it only parse the url. I think the hand-crafted regex is better to match and generate the error.

Reply:
hihihihi sorry :)

I think its better use the  hand-crafted regex to match the url and generate error.